Assets/Scripts/LifeManager.cs

```diff
@@ -35,6 +48,28 @@ public void Die()
 			mPhotonView.RPC("OnDeath", PhotonTargets.MasterClient);
 		}
 
+        public void InflictDamage(int damageAmount)
+        {
+            Assert.IsTrue(damageAmount >= 0);
+            mCurHealth = Mathf.Max(0, mCurHealth - damageAmount);
+
+            mPhotonView.RPC("UpdateLocalHealthBar", PhotonTargets.All, mCurHealth);
```

Comment:
I was looking at some photon stuff and I think there's a better way to do this (as well, it may be worth updating some of our other code). instead of sending to PhotonTargets.All and then guarding against only performing the RPC for the local player on the other side, could we just send this RPC for the owner?

I'm pretty sure we can get the PhotonPlayer from the actor id so we could do something like...
`mPhotonView.RPC("UpdateLocalHealthBar", GetComponent<PhotonView>().owner, mCurHealth)`

this way we only send the update to the correct player and don't bother sending a bunch of unnecessary RPCs